pkg/karmadactl/cmdinit/cmdinit.go

```diff
@@ -97,6 +97,7 @@ func NewCmdInit(parentCommand string) *cobra.Command {
 	// kube image registry
 	flags.StringVarP(&opts.KubeImageMirrorCountry, "kube-image-mirror-country", "", "", "Country code of the kube image registry to be used. For Chinese mainland users, set it to cn")
 	flags.StringVarP(&opts.KubeImageRegistry, "kube-image-registry", "", "", "Kube image registry. For Chinese mainland users, you may use local gcr.io mirrors such as registry.cn-hangzhou.aliyuncs.com/google_containers to override default kube image registry")
+	flags.StringVar(&opts.KubeVersion, "kube-version", "v1.25.2", "Choose a specific Kubernetes version for the control plane.")
```

Comment:
How about `kube-image-tag`? It seems to account for the relation with `kube-image-registry`.

Reply:
I refer to https://github.com/kubernetes/kubernetes/blob/57eb5d631ccd615cd161b6da36afc759af004b93/cmd/kubeadm/app/cmd/options/generic.go#L79

I think these two can be a definition @lonelyCZ

Reply:
> kubeadm is specially to install Kubernetes that is a little different from my scenario. 

Totally agree!

@helen-frank Can you help to describe why we need a new flag? I'm kind of lost from it.

We already have the following flags:
- --karmada-kube-controller-manager-image=''
- --etcd-image=''
- --etcd-init-image='docker.io/alpine:3.15.1'
- --karmada-apiserver-image=''

Reply:
```shell
./karmadactl init --private-image-registry=daocloud.io/atsctoo --kube-version=v1.18.20 --crds=/root/karmada/crds.tar.gz`
```
replace
```shell
./karmadactl init --private-image-registry=daocloud.io/atsctoo --karmada-apiserver-image=daocloud.io/atsctoo/kube-apiserver:v1.18.20 --karmada-kube-controller-manager-image=daocloud.io/atsctoo/kube-controller-manager:v1.18.20 --crds=/root/karmada/crds.tar.gz`
```
For common application scenarios, kube's version is more concerned, and etcd may be neglected, so the scope of tag is limited to kube. @RainbowMango

Reply:
Thanks for your explanation.
I can see there is a slight benefit.
My concern is introducing a new flag(no matter `kube-image-tag` or `--kube-version`) also brings confusion.
It no clear specification of which components would effect by this flag.

Reply:
My opinion is to postpone this feature until we have clear evidence or we have a better alternative.

Reply:
Yes, this parameter is very helpful for me to provide offline installation karmada commands (currently, the platform cannot upgrade kube versions with karmada to adapt, so different versions of kube will be installed in different scenarios)
So, let's take a look at everyone's opinions.  @RainbowMango , can you help me @ some active partners?

Reply:
> currently, the platform cannot upgrade kube versions with karmada to adapt

What do you mean?

Reply:
Some of our existing business platforms have different requirements for kube versions, so there is often a requirement for setting up kube versions when using karmadactl to install karmada, while we are not particularly concerned about others. For the Karmada itself, we use the latest stable version.

Reply:
I think this is a common situation. What do you think

Reply:
Yes, I agree it's a reasonable case. 
Do we have any other alternatives in addition to `kube-image-tag and --kube-version`?

Reply:
At present, there is no better idea. Individuals are more inclined to `kube-version`. `kube-image-tag` is also acceptable.